packages/suite-data/files/translations/en.json

```
@@ -146,8 +147,10 @@
   "TR_DEVICE": "Device",
   "TR_DEVICE_CONNECTED_CONFIRMATION": "I'm not going to disconnect the Trezor before finishing CoinJoin",
   "TR_DEVICE_CONNECTED_TOOLTIP": "Most of the time you can safely disconnect Trezor and the CoinJoin will be paused. However, disconnecting during a critical signing phase can disrupt the CoinJoin round for all participants. You won't lose any money, but you may be banned from some upcoming CoinJoin rounds. All of your funds remain spendable.",
+  "TR_DEVICE_DISCONNECTED_DURING_ACTION_DESCRIPTION": "Your Trezor was disconnected during the backup process. We strongly recommend that you use the factory reset option in Device settings to wipe your device and start the wallet backup process again.",
```

```suggestion
  "TR_DEVICE_DISCONNECTED_DURING_ACTION_DESCRIPTION": "Your Trezor was disconnected during the backup process. We strongly recommend you to use the factory reset option in Device settings to wipe your device and start the wallet backup process again.",
```

Sound better in my opinion